calculator.py

```diff
@@ -0,0 +1,87 @@
+
+
+def add(a, b):
```

Comment:
These are helper funcs not the important ones, helper function name convention is _add(a, b)
There should be two important functions - process_input() and calc()